app/controllers/search_controller.rb

```diff
@@ -1,5 +1,5 @@
 class SearchController < ApplicationController
-
+  skip_before_filter :authenticate_user!
```

Comment:
Browser repo, we do not need skp_before_filter, but in case of search , we need.,
Current version is okay.